app/src/main/res/values/strings.xml

```diff
@@ -73,5 +73,5 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="audio_feedback_pref_key">audioFeedback</string>
     <string name="bit24_pref_key">bit24</string>
     <string name="advanced_mode_pref_key">advancedMode</string>
-
+    <string name="axis_desc_time">Time</string>
```

Comment:
Why did you mark "No modifications done at the end of resource files strings.xml, dimens.xml or colors.xml" as done?

Reply:
Made these changes while updating. Will unmark from PR body template.